src/components/Section/Section.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import './Section.css';
+
+export default function Section() {
+  return (
+    <section id="sectionShared">
+      <div className="title">
+        <FolderInfo />
+      </div>
+    </section>
+  );
+}
+
+function FolderInfo() {
+  const [folderData, setFolderData] = useState();
+  const fetchData = async () => {
+    try {
+      const response = await axios.get('https://bootcamp-api.codeit.kr/api/sample/folder/');
```

Comment:
API의 pathname끝에 슬래시가 있어 응답이 에러로 내려옵니다. 슬래시를 제거해주세요!
```suggestion
      const response = await axios.get('https://bootcamp-api.codeit.kr/api/sample/folder');
```

Reply:
이게 데이터를 받아올 수 없는 문제의 원인이였어요ㅠㅠ 감사합니다 기백님